### PR TITLE
icons, minor state attrs adjustments, None when 0 for timestamp

### PR DESCRIPTION
**Pull request by travisghansen**
Initial stab at icons.

Closes: #72

### CODE REVIEW

File: custom_components/opensprinkler/switch.py [hunk 1]
```diff
@@ -71,13 +71,21 @@ def __init__(self, entry_id, name, controller, coordinator):
...
     @property
     def unique_id(self) -> str:
         """Return a unique, Home Assistant friendly identifier for this entity."""
         return f"{self._entry_id}_{self._entity_type}_controller_operation_enabled"
 
+    @property
+    def icon(self) -> str:
+        """Return icon."""
+        if self._controller.operation_enabled:
+            return "mdi:circle"
```
Comment (vinteo) [inline custom_components/opensprinkler/switch.py:85]: Leave as lightning

Comment (travisghansen) [reply custom_components/opensprinkler/switch.py:85]: @vinteo any interest in the `power-*` icons?

Comment (vinteo) [reply custom_components/opensprinkler/switch.py:85]: how about something a bit more out there... `barley` and `barley-off`

Comment (vinteo) [reply custom_components/opensprinkler/switch.py:85]: or `leak` and `leak-off`

Comment (travisghansen) [reply custom_components/opensprinkler/switch.py:85]: I like the `barley` better than the `leak` :D

Comment (vinteo) [reply custom_components/opensprinkler/switch.py:85]: Let's change it barley and merge this

**Pull request merged**